Changelog.md

```diff
@@ -1,3 +1,8 @@
+### 0.5.6 (unreleased)
+
+Bugfixes:
+ * Library ABIs no longer contain internal functions.
```

Comment:
I think this could be misunderstood. Perhaps "functions with internal types like storage structs". Also should make clear that this is just about the json description of the ABI.